docs/TpchBenchmark.md

```diff
@@ -0,0 +1,453 @@
+# Velox I/O Optimizations and the TpchBenchmark
+## Introduction
+This document is outcome from a cycle of benchmarking to determine best I/O performance against AWS S3 Parquet data for TPCH. It is intended to describe tuning process recommendations and general suggestions for tuning the Velox query engine.
+
+Benchmarking in Velox is made easy with the optionally built TpchBenchmark (velox_tpch_benchmark) executable. To build the benchmark executable _(build/release/velox/benchamrks/tpch/velox_tpch_benchmark)_, patch the code base with a small patch ([Appendix A](#appendix-a-patch-to-build-tpchbenchmark-executable)) then use the following command line to do the build:
+```bash
+$ make release
```

Comment:
+1

Reply:
This was interesting as I thought some changes would be merged related to make. Deepak had a branch with these changes which I why I used them. It will take me a little while to see if this can be done without patching.

Reply:
I will modify the document as if these issues were fixed, we can delay merging but I would like to finish the review process.